src/community/backlog/index.md.njk

```diff
@@ -3,6 +3,7 @@ title: Backlog
 description:
 section: Community
 layout: layout-pane.njk
+order: 0
```

Comment:
non-blocking: should we avoid 0 being the used as the start, it's a familiar concept for devs but not for others?